schema.sql

```diff
@@ -63,4 +63,19 @@ CREATE TABLE visits(
     animal_id INT references animals(id),
     vet_id INT references vets(id),
     date_of_visit DATE
-);
\ No newline at end of file
+);
+
+-- Performance audit
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
+
```

Comment:
- [ ] Great job. As mentioned in the requirements, make sure to have your INSERT INTO in the data.sql file
<img width="1036" alt="image" src="https://user-images.githubusercontent.com/37901469/194896000-ecd36473-65e7-4758-84e7-d34a8a33c5ec.png">